### ISSUE TITLE
Logs spam from `openraft::engine::handler::vote_handler`

### ISSUE THREAD

**Issue by sebadob** [datafuselabs/openraft#1236]
**Describe the bug**
Since `v0.9.14` I see endless log spamming on the `info` level coming from the `openraft::engine::handler::vote_handler`. These seem to happen on follower nodes only. I have not seen them on the leader so far. They look like this:

```
2024-08-27T07:00:02.130351Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:02.207132Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:02.356268Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:02.433548Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:02.582226Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:02.659156Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:02.808812Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:02.885670Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:03.034084Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
2024-08-27T07:00:03.112075Z  INFO openraft::engine::handler::vote_handler: updated last_seen_vote from T1-N1:committed to T1-N1:committed
```

This happens on fresh clusters and on older already existing ones as well.  
When I pin the version to `openraft = "=0.9.13"`, these logs disappear.

**To Reproduce**
Just start up any cluster, I guess. I saw these when working on [Hiqlite](https://github.com/sebadob/hiqlite) and you could simply just `just test` for instance to see these messages, although the tests emit many logs. I also see these from my test deployment on a real environment.

**Expected behavior**
I guess these logs are simply in the wrong log level and should probably be pushed into `trace` or something like this, because they produce a very huge log output in a short amount of time.

**Actual behavior**
See **Describe the bug**, it is pretty simple.

**Env (please complete the following information):**
- `openraft = { version = "0.9.14", features = ["serde", "storage-v2"] }`
- any additional information is probably pointless here, since its just logging

**Additional files:**
This is a screenshot from the log output of a 3 node cluster started inside a K8s test env, nodes 1, 2, 3 from top to bottom. Node 1 became the leader in this case and the followers started logs spamming:

![grafik](https://github.com/user-attachments/assets/e55be3e9-cdd8-4192-a43e-370ffa398c5b)

**Comment by drmingdrmer** [comment]
Thank you for bringing this issue to my attention.

This should be resolved in version [v0.9.15](https://github.com/datafuselabs/openraft/tree/v0.9.15). Please upgrade and let me know if the issue is addressed.

Note that version 0.9.14 has been yanked due to a bug that could potentially cause a brain split. This bug has also been fixed in v0.9.15.

**Comment by sebadob** [comment]
I just tested with `0.9.15` and the issue is gone, thank you!
